packages/kbn-i18n/src/react/provider.tsx

```diff
@@ -45,7 +59,7 @@ export class I18nProvider extends React.PureComponent {
         defaultFormats={i18n.getFormats()}
         textComponent={React.Fragment}
       >
-        {children}
+        {i18n.isPseudoLocale() ? <I18nProvider>{child}</I18nProvider> : child}
```

Comment:
__nit:__ please add a comment here explaining why we need to do that trick for pseudo locales.